core/src/main/java/io/github/tcdl/msb/api/RequestOptions.java

```diff
@@ -69,29 +67,29 @@ public String toString() {
                 + "]";
     }
 
-    public static class Builder<T extends Payload> {
+    public static class Builder {
```

Comment:
We don't really need to have a type param here. It was overlooked in the previous version.